.storybook/webpack.config.ts

```diff
@@ -1,3 +1,6 @@
+const path = require('path');
```

Comment:
Is this is required here as `path` and `mini-css-extract-plugin` seems to be not used at all ?